File: Java/Calcolatrice.java
public class Calcolatrice{
	public static void main(String[]args){
		int a = 4;
		int b = 3;
		int test = 4;
		
		switch(test){
			case 1:{
				System.out.println("hai scelto la somma");
				int c= a+b;
				System.out.println(c);
			}
			break;
			
			case 2:{
				System.out.println("hai scelto la sottrazione");
				int d= a-b;
				System.out.println(d);	
			}
			break;
			
			case 3:{
				System.out.println("hai scelto la moltiplicazione");
				int e= a*b;
				System.out.println(e);
			}
			break;
			
			case 4:{
				System.out.println("hai scelto la divisione");
				double f= a/b;
				System.out.println(f);
			}
			break;
			
			case 5:{
				System.out.println("hai scelto il modulo");
				int g= a%b;
				System.out.println(g);
			}
			break;
			
			default:{
				System.out.println("Non hai scelto nessuna operazione");
			}
		}
	}
}

File: C#/Calcolatrice/Calcolatrice.cs
using System;

namespace Calcolatrice
{
    public class Calcolatrice{
			
    public static void Main(){
	Console.WriteLine("-------------------------------------");
	Console.WriteLine("------------ Calcolatrice -----------");
	Console.WriteLine("-------------------------------------");
			
	Console.WriteLine("0x00 -- Indice");
	Console.WriteLine("1 --> Somma ");
	Console.WriteLine("2 --> Sottrazione");
	Console.WriteLine("3 --> Moltiplicazione");
	Console.WriteLine("4 --> Divisione");
			
	Console.WriteLine("\nInserisci il numero dell'operazione: ");
	String op = Console.ReadLine();
			
	int val;
			
	switch(op){
	    case 1:
		Console.WriteLine("Inserisci il numero di valori da sommare ");
		val = Console.Read();
				
		for(int i = 0; i < val; i++){
						
		}
				
		somma();
		break;
					
	}
				
				
		}
		
		public int somma(int v1, v2){
			return v1 + v2	
		}
		
		public int sottrazione(int v1, int v2){
			return v1 - v2;
		}
		
		public int moltiplicazione(int v1, int v2){
			return v1 * v2;	
		}
		
		public int divisione(int v1, int v2){
			return v1 / v2;	
		}
	}
}

